ledger/internal/eval.go

```diff
@@ -934,6 +934,8 @@ func (eval *BlockEvaluator) transactionGroup(txgroup []transactions.SignedTxnWit
 	var groupTxBytes int
 
 	cow := eval.state.child(len(txgroup))
+	defer childPool.Put(cow)
+
```

Comment:
I think there should be a pair `recycle`/`emptyNest`/whatever function on child cow object that operates the pool, so pool.Get and pool.Put leaves in the same file / abstraction.